__tests__/app.test.js

```diff
@@ -450,3 +450,45 @@ describe("app", () => {
     });
   });
 });
+
+describe("app", () => {
+  describe("GET /api/users", () => {
+    it("200: GET - Responds with an array of objects", () => {
```

Comment:
I would remove this test as we are implicitly checking that this is an array in the next test